lib/dynflow/web/console.rb

```diff
@@ -49,6 +50,21 @@ class Console < Sinatra::Base
         erb :worlds
       end
 
+      get('/export') do
+        set_download_headers('index.tar.gz')
+        Exporters::Tar.full_html_export(find_execution_plans(true, false), self)
```

Comment:
Those were meant as an example how to use the tar exporter, maybe removing the index-like export api endpoints and having a separate PR for a well-thought api with filtering would be better